modules/pruning-images.adoc

```diff
@@ -36,9 +36,10 @@ spec:
   startingDeadlineSeconds: 60 <9>
   successfulJobsHistoryLimit: 3 <10>
   failedJobsHistoryLimit: 3 <11>
+  logLevel: TraceAll <12>
```

Comment:
spec:
  logLevel: Normal

The default value is Normal, could we change to Normal?

Reply:
Certainly! Updating now. Thanks!